develop-docs/application/dynamic-sampling/extrapolation.mdx

```diff
@@ -0,0 +1,95 @@
+
+To account for this fact, Sentry offers a feature called Extrapolation. Extrapolation smartly combines the data that was ingested to account for different sample rates in different parts of the application. However, low sample rates may cause the extrapolated data to be less accurate than if there was no sampling at all.
+
+So how does one handle this type of data, and when is extrapolated data accurate and expressive? Let’s start with some definitions: 
+
+- Accuracy refers to data being correct. For example, the measured number of spans corresponds to the actual number of spans that were executed. As sample rates decrease, accuracy also goes down, because minute random decisions can influence the result in major ways, in absolute numbers. Also, when traffic is low and 100% of data is sampled, the system is fully accurate despite aggregates being affected by inherent statistical uncertainty.
+- Expressiveness refers to data being able to express something about the state of the observed system. For example, a single sample with specific tags and a full trace can be very expressive, and a large amount of spans can have very misleading characteristics. Expressiveness therefore depends on the use case for the data.
+
+Affected Product Surface
+
+1. Explore
+2. Alerts
+3. Dashboards
+4. Requests
+5. Profiles
+
+### **Modes**
+
+There are two modes that can be used to view data in Sentry: extrapolated mode and sample mode.
+
+- Extrapolated mode extrapolates the ingested data as outlined above.
+- Sample mode does not extrapolate and presents exactly the data that was ingested.
+
+Depending on the context and the use case, one mode may be more useful than the other. 
+
+There is currently no way for Sentry to automatically switch from the extrapolated mode into sample mode based on query attributes, therefore the transition needs to be triggered by the user. However, Sentry can nudge the user, based on observed characteristics of a query, to switch from one mode to another. One example for this is when an ID column is detected: extrapolated aggregates for high-cardinality and low-volume ID columns are usually not very useful, because they may refer to a highly exaggerated volume of data that is not extrapolated correctly due to the high-cardinality nature of the column in question.
```

Comment:
I think we should describe the desired state and reduce focus on current limitations. Topics we can address independently are:
- Which kinds of queries do not make sense to extrapolate in principle?
- Should sample mode be a transient view option or a persistent setting on a certain scope?
- How can a user opt out of extrapolation - should that be explicit, implicit, and how does the system support this?